src/tracer.js

```diff
@@ -64,7 +64,9 @@ export default class Tracer {
     options: any = {}
   ) {
     this._tags = options.tags || {};
-    this._tags[constants.JAEGER_CLIENT_VERSION_TAG_KEY] = 'Node-3.14.4';
+
+    // $FlowIgnore - stop complaining about property `env` not found
+    this._tags[constants.JAEGER_CLIENT_VERSION_TAG_KEY] = `Node-${process.env['PACKAGE_VERSION']}`;
```

Comment:
I don't understand this - who is going to provide this env variable when the tracer is used in the end-user's application?

Reply:
or is it going to be resolved at transpile-time?

Reply:
babel-plugin-transform-inline-environment-variables changes the built file to be `'Node-' + '3.14.4'` (whatever the version is when babel transpiles). it is defined by the makefile.

Reply:
I packed the module (`make node-build && yarn pack`)and installed the tar in other folder and it has the version at build time in the dist:
![image](https://user-images.githubusercontent.com/3471184/53445592-dfb1b880-39de-11e9-91cb-2da49441c899.png)